### PR TITLE
Added methods getters/setters for age and drinking age,method for drinking age

### PR DESCRIPTION
**Pull request by cpalmerbright**

### CODE REVIEW

File: models/human.rb [hunk 1]
```diff
@@ -1,29 +1,17 @@
...
-  # attr_accessor :job_title
-  def job_title=(value)
-    @job_title = value
+    @name.split().reverse.join(', ')
   end
 
-  def job_title
-    @job_title
+  def can_drink?
+    return @age >= @drinking_age
```
Comment (nigelr) [inline models/human.rb]: you don't need to include the return here, methods by default return the last thing evaluated

File: specs/models/human_spec.rb [hunk 2]
```diff
@@ -39,9 +39,61 @@
...
+
+  describe '.can_drink?' do
+    it 'returns true if instance of human is younger than drinking_age' do
+      human = Human.new('')
+      human.age = 17
+      human.drinking_age = 18
+      expect(human.can_drink?).to be_falsey
+    end
+  end
+
```
Comment (nigelr) [inline specs/models/human_spec.rb]: formatting only, don't leave a line between ends

File: specs/models/human_spec.rb [hunk 3]
```diff
@@ -39,9 +39,61 @@
...
+    it 'returns true if instance of human is the same age as drinking_age' do
+      human = Human.new('')
+      human.age = 21
+      human.drinking_age = 21
+      expect(human.can_drink?).to be_truthy
+    end
+  end
+
+  describe '.can_drink?' do
+    it 'returns true if instance of human is younger than drinking_age' do
```
Comment (nigelr) [inline specs/models/human_spec.rb]: your description is wrong here 
This is known as a copy and paste error...

File: specs/models/human_spec.rb [hunk 4]
```diff
@@ -39,9 +39,61 @@
...
+  describe '.can_drink?' do
+    it 'returns true if instance of human is older than drinking_age' do
+      human = Human.new('')
+      human.age = 20
+      human.drinking_age = 18
+      expect(human.can_drink?).to be_truthy
+    end
+  end
+
+  describe '.can_drink?' do
```
Comment (nigelr) [inline specs/models/human_spec.rb]: you should only `describe '.can_drink?' do` once and group all the `it`s inside it.

File: specs/models/human_spec.rb [hunk 5]
```diff
@@ -39,9 +39,61 @@
...
+    it 'returns true if instance of human is the same age as drinking_age' do
```
Comment (nigelr) [inline specs/models/human_spec.rb]: What if `age` or `drinking_age` was not set or was set to nil

**Review by nigelr — CHANGES_REQUESTED** [review]

File: specs/models/human_spec.rb (hunk 5 above)
Comment (cpalmerbright) [reply specs/models/human_spec.rb]: is there a standard way to deal with these situations? 
Should I have the can_drink? method also return nil, or should I have it return false automatically, or should it return a request for the user to set those values first?

File: models/human.rb [hunk 6]
```diff
@@ -1,29 +1,21 @@
...
-  # attr_accessor :job_title
-  def job_title=(value)
-    @job_title = value
-  end
+  #@drinking_age = 18
 
-  def job_title
-    @job_title
+  def can_drink?
+    @drinking_age ||= 18
```
Comment (nigelr) [inline models/human.rb:17]: might this be a situation where having `def drinking_age` getter may be better and we default it there ensuring any other method that uses it defaults to the same value.
ps. same for `age`

Comment (nigelr) [reply models/human.rb:17]: pps. and you would need to test for this as well

**Review by nigelr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**